invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/index.js

```diff
@@ -151,6 +151,7 @@ const LogoUploader = ({ community, defaultLogo, hasLogo, onError }) => {
               fluid
               wrapped
               rounded
+              className="community-logo tall"
```

Comment:
I am not sure what is the reason to add a separate `tall` class to limit the height ? why not to do it simply with `community-logo` ?

Reply:
because there is more space in the settings tab to show the logo. the height will be too big for the list items